src/python/WMComponent/RucioInjector/Database/Oracle/GetCompletedBlocks.py

```diff
@@ -0,0 +1,15 @@
+"""
+_GetCompletedBlocks_
+
+Oracle implementation of RucioInjector.GetComletedBlocks
+
+"""
+
+from __future__ import division
+from __future__ import print_function
+
+from WMComponent.RucioInjector.Database.MySQL.GetCompletedBlocks import GetCompletedBlocks as MySQLBase
```

Comment:
I'd rather keep it as it is, because I noticed this `MySQLBase` name is used as a kind of a standard naming convention in all Oracle modules for the underlying `MySQL` Class which they inherit from.

Reply:
No problem!
I just checked a few random DAOs and really there is no common sense on how to import it. The "Base" word looks misleading to me though, since it's not a base class but just the specific MySQL implementation.